src/widgets/medications/medication-level-two.component.tsx

```diff
@@ -127,29 +127,27 @@ export default function MedicationLevelTwo(props: MedicationsOverviewProps) {
                               action={"REVISE"}
                               inProgress={true}
                             />
-                            <td>
-                              <MedicationButton
-                                component={MedicationOrderBasket}
-                                name={"Medication Order Basket"}
-                                label={"DISCONTINUE"}
-                                orderUuid={medication.uuid}
-                                drugName={medication.drug.name}
-                                action={"DISCONTINUE"}
-                                inProgress={true}
-                              />
-                            </td>
-                            <td style={{ textAlign: "end" }}>
-                              <Link
-                                to={`/patient/${patientUuid}/chart/medications/${medication.uuid}`}
+                            <MedicationButton
+                              component={MedicationOrderBasket}
+                              name={"Medication Order Basket"}
+                              label={"DISCONTINUE"}
+                              orderUuid={medication.uuid}
+                              drugName={null}
+                              action={"DISCONTINUE"}
```

Comment:
I thought there were named constants somewhere for this actions

Reply:
Corrected